apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -974,21 +974,25 @@ const Ticket = new GQLListSchema('Ticket', {
     },
     kmigratorOptions: {
         indexes: [
+            // NOTE: popular filter on /ticket page
             {
                 type: 'BTreeIndex',
                 fields: ['unitName'],
                 name: 'ticket_unitName',
             },
+            // NOTE: used on /ticket page for "Own Tickets" fitler
             {
                 type: 'BTreeIndex',
-                fields: ['assignee', 'executor', 'organization'],
-                name: 'ticket_assignee_exec_org',
+                fields: ['organization', 'assignee', 'executor', 'deletedAt'],
+                name: 'ticket_org_ass_execut_deletedAt',
```

Comment:
```suggestion
                name: 'ticket_org_assign_exec_deletedAt',
```

<img width="315" alt="image" src="https://github.com/user-attachments/assets/719f5c4c-ffa2-4b21-ad08-36073c62667f">

Reply:
thanks, fixed it